docs/ops/arithmetic/Sqrt_1.md

```diff
@@ -25,13 +23,30 @@
 *Sqrt* does the following with the input tensor *a*:
```

Comment:
Types should be the end of the spec. We can move this formula to be part of short description or add it in detailed description section

Reply:
I've added detailed description and moved the math formula there